packages/okta-angular/package.json

```diff
@@ -40,7 +40,7 @@
     "license": "Apache-2.0",
     "dependencies": {
         "@okta/configuration-validation": "^0.4.1",
-        "@okta/okta-auth-js": "^2.11.2",
+        "@okta/okta-auth-js": "3.0.0",
```

Comment:
Why exact?

Reply:
Because I typed it in manually. I'll update to a flexible semver